generate-release-notes.py

```diff
@@ -28,14 +47,22 @@ def issues_in_milestone():
         if m.title == MILESTONE:
             # Found the milestone in this repo - look for issues (but only
             # ones that have been closed!)
+            print(f"  found milestone {m.title}")
             issues = []
-            for i in repo.get_issues(milestone=m, state="closed", labels=['release-note-required']):
-                pr = i.as_pull_request()
+            milestone_issues = repo.get_issues(
```

Comment:
Why this change to do this in two lines?

We shouldn't be making changes that don't have a functional impact without good reason - makes it harder to review, and muddles the history up in git.

Reply:
this is from running black formatter

Reply:
If we're going to start to adhere to a style guide we probably want to include it as part of CI to prevent non-conformant changes getting merged.

That said, eventually this should all be Go code if I had my way, so maybe not that important to do now so long as we can see that on the horizon...

Reply:
I was templted to switch to Go after the latest round of reviews but I wanted to get this script meged with a good working base. I will like to park enforce a format to when we swich to Golang sicne that is part of my objective this quarter

Reply:
Golang is shipped with a linter - `gofmt` that is standard and we run against all of our code. It will happen naturally when we switch this to Go.